packages/near-membrane-base/src/blue.ts

```diff
@@ -284,9 +324,9 @@ export function blueProxyFactory(env: MembraneBroker) {
             try {
                 red = ReflectApply(target as RedFunction, redThisArg, redArgArray);
             } catch (e) {
-                throw env.getBlueValue(e);
```

Comment:
there were a bunch of unnecessary dot notation when getBlueValue was actually locally.